src/Purestone/Game.hs

```diff
@@ -4,19 +4,48 @@
 -- This source code is licensed under the MIT licence found in the           --
 -- LICENSE file in the root directory of this source tree.                   --
 -------------------------------------------------------------------------------
-module Purestone.Game ( setupGame, processMove ) where
+module Purestone.Game ( setupGame, processMove, draw, resetPoints,
```

Comment:
Looks good, the only thing is do we need to expose any functions other than `setupGame` and `processMove`, if those helper functions are only used inside this file we don't need to list them here

Reply:
Yeah, I added the draw and the functions which effect the points as cards may call them, though I wasn't sure. Start turn could be removed though as I don't believe that is needed outside of it.